docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -81,15 +80,6 @@ is dynamic and does not require a restart.
       For example, a rolling restart is applicable when changing a WebLogic Server `stuck thread timer interval` property. See [Restart all the servers in the domain]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#restart-all-the-servers-in-the-domain" >}}).
 
 
-#### Changing the custom domain configuration overrides
-
```

Comment:
Doesn't this still belong? There are still cases where a change may require a restart.
This can point to the new 'supported changes' section you added?

Reply:
It's currently described as always requiring a full domain restart. Under what conditions would it still require that?

Reply:
Changing network related overrides?  I thought you documented a set of them in the new introspect document...